pandas/indexes/base.py

```diff
@@ -1392,7 +1392,7 @@ def __getitem__(self, key):
         else:
             return result
 
-    def append(self, other):
+    def append(self, other, dropna=True):
```

Comment:
The problem here is that I need to call the derived `append()` in `Multi.py` and the `dropna` in `Multi.append` cannot be omitted. The possible way to remove this `dropna` as well as the one [here](https://github.com/pydata/pandas/pull/14246/files#diff-e6d27b1c748bfee53e0bdc8621429a51L1272) is: have an parameter `dropna` in `MultiIndex.__init__()`. I am not sure how you like this idea. I would be appreciated if you could tell me your thoughts on it. Thanks!